compose/config/config.py

```diff
@@ -725,6 +729,7 @@ def merge_service_dicts(base, override, version):
         'external_links',
         'ports',
         'volumes_from',
+        'tmpfs',
```

Comment:
Since this can be a list or a string it needs to be in the list with `dns` and `dns_search`.